### ISSUE TITLE
Kanban and Custom Fields Module: Custom fields are displayed twice 

### ISSUE THREAD

**Issue by dev4223** [freescout-helpdesk/freescout#3390]
PHP version: 8.1
FreeScout version: v1.10.56

We have both modules installed and did setup some custom fields. When the option "Show In Conv. List" is ticked on for a field, it is shown twice in kanban view. It is shown only once in normal mailbox list view.
If all custom fields are filled for a ticket, first comes a line of all the custom fields in order and the the same row of custom fields is shown afterwards again.

When I tick the option off, the custom field is only shown once in kanban view and is not visible in list view.

![230913-custom fields shown twice](https://github.com/freescout-helpdesk/freescout/assets/400822/2b49db64-0330-4b2f-a82c-86380ab45ce8)

**Comment by freescout-helpdesk** [comment]
So would you like to see all the custom fields in Kanban view and ignore "Show In Conv. List" option? Or you want to see only those custom fields which are marked as "Show In Conv. List"?

**Comment by dev4223** [comment]
The second rule would make more sense for me: Show the custom fields in overviews like kanban and list view according to the setting of  "Show In Conv. List" option.
Another nice-to-have would be to add a second switch, so one can set the options for kanban and list view seperately. The switch for kanban view is only shown, if kanban module is activated.

**Comment by freescout-helpdesk** [comment]
Kanban Module v1.0.18 now shows only custom fields marked as "Show in Conv. List".

**Comment by dev4223** [comment]
Perfect, looks good! Thank you very much!
